source/slang/slang-capability.cpp

```diff
@@ -988,6 +988,104 @@ void CapabilitySet::canonicalize()
     m_conjunctions.sort();
 }
 
+CapabilitySet CapabilitySet::getTextualTargetsThisIsMissingFromOther(const CapabilitySet& other)
```

Comment:
Why `textualTarget`? Shouldn't it handle non textual targets just fine?

Reply:
You are correct, my naming would be wrong here (intended behavior is for targets*).